IAmLegend/Assets/Scripts/GameManager.cs

```diff
@@ -31,8 +41,12 @@ void Start()
 
     void Update()
     {
+        // Update the timer
+        timer += Time.deltaTime;

```

Comment:
We might need to tune the duration of the game so it might a good idea to add a multiplier here called the timeMultipler. We can then use it to make the game slower or faster as we tune the level